hail_search/queries/snv_indel.py

```diff
@@ -90,10 +98,25 @@ def _get_gnomad_af_prefilter(self, frequencies=None, pathogenicity=None, **kwarg
         if af_cutoff is None:
             return False
 
-        if self._get_clinvar_path_filters(pathogenicity):
-            af_cutoff = max(af_cutoff, PATH_FREQ_OVERRIDE_CUTOFF)
+        af_cutoff_field = self._get_af_prefilter_field(af_cutoff)
+        if af_cutoff_field is None:
+            return False
+
+        af_filter = True if af_cutoff_field is True else lambda ht: ht[af_cutoff_field]
+
+        if af_cutoff < PATH_FREQ_OVERRIDE_CUTOFF:
+            clinvar_path_ht = self._get_loaded_clinvar_prefilter_ht(pathogenicity)
+            if clinvar_path_ht is not False:
+                path_cutoff_field = self._get_af_prefilter_field(PATH_FREQ_OVERRIDE_CUTOFF)
+                non_clinvar_filter = lambda ht: hl.is_missing(clinvar_path_ht[ht.key])
+                if af_filter is not True:
+                    non_clinvar_filter = lambda ht: non_clinvar_filter(ht) & af_filter(ht)
+                af_filter = lambda ht: ht[path_cutoff_field] | non_clinvar_filter(ht)
```

Comment:
For clinvar path and likely path variants, seqr will return variants with up to 5% frequency even if the filter is set lower than that. To address this in the prefilter, we set it so if the clinvar path filter existed we just use the higher 5% cutoff for the prefilter for all variants. However, since we actually also have a clinvar prefilter table that we are already reading in to help with the quality filtering, we can add logic here so variants in clinvar are filtered out if they have greater than 5% frequency, and variants not in clinvar are prefiltered to the actual threshhold